src/content/docs/infrastructure/host-integrations/installation/secrets-management.mdx

```diff
@@ -743,3 +743,67 @@ variables:
 
 proxy: ${obfuscated_proxy}
 ```
+
+## Custom Commands [#custom-commands]
+
+<Callout variant="important">
+  The `command` configuration requires Infrastructure Agent `1.41.0` or greater.
+  Make sure the agent has the required permissions to execute the configured `command` based on your environment requirements.
+</Callout>
+
+You can load agent or integrations configurations from custom commands available in the same host.
+The Infrastructure Agent will detect when configurations should initially be loaded from a `command` output and keep values updated based on a `ttl` refresh time.
+
+The expected output format for the commmand is either JSON or String:
+
+### JSON
+The following JSON structure is expected as the main output (stdout) from the defined `command`: 
+```
+{
+  "data": {
+    "myKey": "myValue"
+  }, 
+  "ttl": "24h"
+}
+```
+- A top-level `data` field is required.
+- A top-level `ttl` field is required.
+
+It can then be used in any configuration file using the `${myVariable.key}` notation.
+In the following example, a `my-custom-auth` command is executed with a `domain` parameter then loading the resulting `token` as an agent configuration: 
+
+```
+variables:
+  myToken:
+    command:
+      path: '/path/to/my-custom-auth-json'
+      args: ["--domain", "myDomain", "--other_param", "otherValue"] 
+      ttl: 24h
+      
+# my-custom-auth output example: {"data":{"token":"XXXXYYYYZZZZ"}, "ttl": "24h"}
+# the `token` result can be included as ${myToken.token} in any configuration.
+  
+http:
+  headers: 
+    "Proxy-Authorization": ${myToken.token}
+```
+
+### String (text/plain)
+If the command output is a plain string, then it can be referenced in the configuration using the ${myVar} notation.
```

Comment:
this variable seems to be breaking the build

```suggestion
If the command output is a plain string, then it can be referenced in the configuration using the `${myVar}` notation.
```